radiomics/featureextractor.py

```diff
@@ -284,6 +284,9 @@ def enableFeatureClassByName(self, featureClass, enabled=True):
     """
     Enable or disable all features in given class.
     """
+    if not featureClass in self.getFeatureClassNames():
```

Comment:
Thanks! I admit I forgot to follow up and check the tests after submitting the PR.